examples/widgets/smooth_resize.dart

```diff
@@ -26,29 +26,28 @@ class SmoothBlock extends StatefulComponent {
 class CardTransition extends StatelessComponent {
   CardTransition({
     this.child,
-    this.performance,
+    this.animation,
     this.x,
     this.opacity,
     this.scale
   });
 
   final Widget child;
-  final Performance performance;
-  final AnimatedValue<double> x;
-  final AnimatedValue<double> opacity;
-  final AnimatedValue<double> scale;
+  final Animation animation;
+  final Tween<double> x;
+  final Tween<double> opacity;
+  final Tween<double> scale;
 
   Widget build(BuildContext context) {
-
-    return new BuilderTransition(
-      performance: performance,
-      variables: <AnimatedValue<double>>[x, opacity, scale],
+    return new AnimationWatchingBuilder(
+      watchable: animation,
       builder: (BuildContext context) {
+        double currentScale = scale.evaluate(animation);
         Matrix4 transform = new Matrix4.identity()
-          ..translate(x.value)
-          ..scale(scale.value, scale.value);
+          ..translate(x.evaluate(animation))
+          ..scale(currentScale, currentScale);
         return new Opacity(
-          opacity: opacity.value,
+          opacity: opacity.evaluate(animation),
```

Comment:
Another option would be to convert the incoming arguments in the constructor into Evaluatable<double>s.